source/js/viewer-core.js

```diff
@@ -404,7 +404,8 @@ function ViewerCore(element, options, publicInstance)
             var options = {
                 viewport: viewerState.viewport,
                 outerElement: viewerState.outerElement,
-                innerElement: viewerState.innerElement
+                innerElement: viewerState.innerElement,
+                enableCrossOrigin: settings.options.enableCrossOrigin
```

Comment:
Can you confirm that this is not actually supposed to be set through `viewerState`?